asmcli/commands/help.sh

```diff
@@ -183,6 +168,15 @@ FLAGS:
      --use_managed_cni                Deprecated. Managed CNI now defaults to be on.
      --managed                        Provision a remote, managed control plane
                                       instead of installing one in-cluster.
+                                      The channel matches with the GKE version:
+                                        GKE Channel  |  ASM Channel
+                                        -------------|----------------
+                                        Rapid        |  Rapid
+                                        Regular      |  Regular
+                                        Stable       |  Stable
+                                        Static       |  Regular
```

Comment:
Maybe it's clearer to call this "No channel"?

Also, it has been suggested that No-channel should be Stable. At least for GSM data plane we've already decided to shift this way. Let's consider whether it might make sense to change this here.

Reply:
Never mind the comment about no-channel => stable. That would change existing asmcli behavior, and we don't want that.